### PR TITLE
System-wide *.onion support for Prometheus VM

### PR DESCRIPTION
**Pull request by darkk**
That's infra part of _Monitoring public OONI `*.onion` endpoints_ for #239.

### CODE REVIEW

**Comment by darkk** [comment]
That's actually deployed, so, merging.

**Comment by SuperQ** [comment]
Ohh, neat.

File: ansible/deploy-tor.yml [hunk 1]
```diff
@@ -3,35 +3,9 @@
 
 - hosts: have_tor
   gather_facts: true # to fetch `jessie` codename
+  roles:
+    - role: tor-bin
```
Comment (bassosimone) [inline ansible/deploy-tor.yml]: So, this is a subroutine call in the Ansible world, eh?

Comment (hellais) [reply ansible/deploy-tor.yml]: It's more like a package of stuff that allow you to configure or install some unit of functionality on a system.
See: https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html

Comment (darkk) [reply ansible/deploy-tor.yml]: Yes, I usually treat it as a subroutine call :)
Making a "good" and "reusable" package is a complex exercise and sometimes it leads to (possibly, unnecessary) levels of abstractions.

**Comment by darkk** [comment]
@SuperQ yep. I've tested it within another infra for a while and it works quite well, it also helped to understand some issues regarding hidden service management in [dynamic](https://labs.ripe.net/Members/ramakrishna_padmanabhan/reasons-dynamic-addresses-change) environment.

I hope to allocate some time in May for a blog post on `*.onion` scraping if you're still interested :-)

File: ansible/deploy-tor.yml (hunk 1 above)
Comment (bassosimone) [reply ansible/deploy-tor.yml]: Thanks for the docs! Yeah, I figured it's a complex exercise!

**Pull request merged**